web/core/components/global/product-updates/modal.tsx

```diff
@@ -71,7 +71,7 @@ export const ProductUpdatesModal: FC<ProductUpdatesModalProps> = observer((props
                   },
                 }}
                 fileHandler={{
-                  getAssetSrc: () => "",
+                  getAssetSrc: () => Promise.resolve(""),
```

Comment:
This is fine, but just adding async to the function definition will also fix this